tests/testthat/test-social-smells.R

```diff
@@ -0,0 +1,112 @@
+test_that("Using Perceval version other than 0.12.24 will fail for parse_mbox", {
+  perceval_path <- "../../../tools/bin/perceval-incompatible-version"
+  mbox_path <- "../../../../rawdata/mbox/project.mbox"
```

Comment:
Hi Carlos,
I split the test "Using Perceval version other than 0.12.24 will fail for parse_mbox" into two separate tests and fixed the hardcoding issue.

```r
test_that("Perceval version is 0.12.24", {
  tools_path <- "../../tools.yml"
  tool <- yaml::read_yaml(tools_path)
  perceval_path <-tool[["perceval"]]
  perceval_version <- system2(perceval_path, args="--version", stdout=TRUE, stderr=FALSE)
  expect_equal(perceval_version, "perceval 0.12.24")
})
```

```r
test_that("Perceval path works for parse mbox", {
  tools_path <- "../../tools.yml"
  conf_path <- "../../conf/thrift.yml"
  tool <- yaml::read_yaml(tools_path)
  conf <- yaml::read_yaml(conf_path)
  perceval_path <-tool[["perceval"]]
  mbox_path <- paste("../", conf[["mailing_list"]][["mbox"]], sep="")
  expect_no_error(parse_mbox(perceval_path, mbox_path))
})
```

How does this look?

Also, just wanted to confirm, the "rawdata" folder we see in the config files is only one parent directory outside of Kaiaulu, right?

Reply:
It should be one level above Kaiaulu, yes.

The Perceval test looks fine. 

The second test I am not sure what you are really testing. Try to refine it further, so that you can use expect_equal too.